docs/docs/development.md

```diff
@@ -1,6 +1,24 @@
-# Local Testing
+# Testing
 
-## Initialize kubernetes cluster
+## Local Testing
+
+Local testing is a great way to test the components of QHub. While it
+does test most of everything is does not test cloud provisioned
+components such as the managed kubernetes cluster, vpcs, managed
+container registries etc.
+
+This guide assumes that you have the QHub Cloud repository downloaded
+and you are at the root of the repository.
+
+### Dependencies
+
+In order to develop with QHub you will need to download the following
+dependencies and have them available in your path.
+
+ - [terraform](https://www.terraform.io/downloads.html)
+ - [minikube](https://v1-18.docs.kubernetes.io/docs/tasks/tools/install-minikube/)
```

Comment:
As a newbie, I need a bit more detail to understand this part.
It seems like to have MiniKube I will need to have Docker installed. Do I also need Docker Engine?

What do you recommend for the VMs? KVM? VirtualBox? Are VMs even necessary at all?

Reply:
Also, the starting point was not 100% clear for me.

Do I start on a clean virtual env in an empty folder, install minikube and then run minikube commands, use `python -m qhub init` to generate the initial config file, and go from there?

Reply:
> It seems like to have MiniKube I will need to have Docker installed. Do I also need Docker Engine?

In general not necessarily, you can use hyperkit as well, but the instructions for that might change a lot so I would recommend you have docker installed.

> What do you recommend for the VMs? KVM? VirtualBox? Are VMs even necessary at all?

Docker will take care of that, when you run the minikube start command as mentioned below

Reply:
Alright. So Docker is a necessary requirement so VMs are not. Understood! 🎖️ 
For now, I will stay away from HyperKit but will have a look at it. Thanks :)